docs/README.md

```diff
@@ -8,9 +8,13 @@
 
 `Caravel` is a local web GUI for [looper](https://looper.readthedocs.io/). Caravel gives you a slick web interface to interact with your projects formatted as [PEPs](http://pepkit.github.io). You can use it to submit jobs to any cluster resource manager, monitor jobs, summarize results, and browse project summary web pages. `caravel` basically builds on top of `looper`, but uses the lightweight `flask` microframework to give you a web interface, making it even easier to manage your jobs for those who like pointy-clicky.
 
+Caravel is currently in *alpha*. It is functional but not very useful yet. We expect to release a useful version in February 2019.
```

Comment:
Should change to March?

Reply:
No, it's still February... February 32nd.